swift_package/BrainFlow/Tests/BrainFlowTests/BrainFlowCITests.swift

```diff
@@ -0,0 +1,404 @@
+//
+//  BrainFlowCItests.swift
+//  functions for Continuous Integration (CI) testing of the Swift Bindings for BrainFlow
+
+//  Created by Scott Miller for Aeris Rising, LLC on 9/23/21.
+
+
+import Foundation
+import XCTest
+import BrainFlow
+
+class BrainFlowCItests: XCTestCase {
+    func testGetDataFromBoard() throws {
```

Comment:
the idea for these tests was to put them in brainflow docs as simple examples which users can copypaste. In docs they are added as links to files with code, its why they should be in different files

Reply:
ah, I see, you also added them separately, good job, its nice